nodebrew

```diff
@@ -372,6 +372,7 @@ sub _cmd_exec {
     my $command = join ' ', @$args;
 
     system $command;
+    exit $? >> 8;
```

Comment:
I can't understand meaning of `>> 8` :disappointed_relieved:

I think the following code is good enough.

```perl
my $status = system $command;
exit $status;
```